README.md

```diff
@@ -1,4 +1,55 @@
+![puta](https://github.com/PactCoffee/js-ui-challenge/raw/master/puta.png)
+
 Pact's front end challenge
 ==========================
 
-TBA
+The general idea
+----------------
+
+We have a sign up flow affectionately known as "The Funnel". It's primary use is to sign up customers to our service. They enter their information, it gets sent to the server. **The challenge is to build a very simple version of [our current funnel](http://pactcoffee.com/funnel/intro)**. Sounds easy enough, right?
+
+
+But wait! There's more...
+-------------------------
+
+1. The funnel is separated into "stages". Each stage collects certain information from the user. For this challenge, those stages are:
```

Comment:
As someone try to complete the task. it would be great to have a list of data to be collected. 

i.e. 
- first_name
- email
- coffee preparation
- address_line1, 2, city. postcode